modules/builtin/src/backend/prompts/number.ts

```diff
@@ -0,0 +1,53 @@
+import { IO, Prompt, PromptConfig } from 'botpress/sdk'
+import { max, min } from 'lodash'
+import moment from 'moment'
+
+class PromptNumber implements Prompt {
+  private _min: boolean
+  private _max: boolean
+
+  // Those are the actual values of the configured prompt on the workflow
+  constructor({ min, max }) {
+    this._min = min
+    this._max = max
+  }
+
+  // This method is provided with the incoming event to extract any necessary information
+  extraction(event: IO.IncomingEvent): { value: number; confidence: number } | undefined {
```

Comment:
I think we could move that to the sdk, ex ExtractionResult, we're gonna reuse it often. 
Comments can also be removed